docs/use/rf.md

```diff
@@ -1,9 +1,142 @@
+Registering protocol [57] "Kerui PIR / Contact Sensor"
+Registering protocol [58] "LaCrosse TX Temperature / Humidity Sensor"
+Registering protocol [59] "LaCrosse TX141-Bv2, TX141TH-Bv2, TX141-Bv3, TX141W, TX145wsdth, (TFA, ORIA) sensor"
+Registering protocol [60] "LaCrosse/ELV/Conrad WS7000/WS2500 weather sensors"
+Registering protocol [61] "LaCrosse WS-2310 / WS-3600 Weather Station"
+Registering protocol [63] "Markisol, E-Motion, BOFU, Rollerhouse, BF-30x, BF-415 curtain remote"
+Registering protocol [64] "Maverick et73"
+Registering protocol [66] "Missil ML0757 weather station"
+Registering protocol [68] "Nexus, FreeTec NC-7345, NX-3980, Solight TE82S, TFA 30.3209 temperature/humidity sensor"
+Registering protocol [70] "Opus/Imagintronix XT300 Soil Moisture"
+Registering protocol [71] "Oregon Scientific Weather Sensor"
+Registering protocol [72] "Oregon Scientific SL109H Remote Thermal Hygro Sensor"
+Registering protocol [74] "Philips outdoor temperature sensor (type AJ3650)"
+Registering protocol [75] "Philips outdoor temperature sensor (type AJ7010)"
+Registering protocol [76] "Prologue, FreeTec NC-7104, NC-7159-675 temperature sensor"
+Registering protocol [77] "Quhwa"
+Registering protocol [79] "Rubicson Temperature Sensor"
+Registering protocol [80] "Rubicson 48659 Thermometer"
+Registering protocol [81] "Conrad S3318P, FreeTec NC-5849-913 temperature humidity sensor"
+Registering protocol [82] "Silvercrest Remote Control"
+Registering protocol [83] "Skylink HA-434TL motion sensor"
+Registering protocol [84] "Wireless Smoke and Heat Detector GS 558"
+Registering protocol [85] "Solight TE44/TE66, EMOS E0107T, NX-6876-917"
+Registering protocol [86] "Springfield Temperature and Soil Moisture"
+Registering protocol [87] "TFA Dostmann 30.3221.02 T/H Outdoor Sensor"
+Registering protocol [88] "TFA Drop Rain Gauge 30.3233.01"
+Registering protocol [89] "TFA pool temperature sensor"
+Registering protocol [90] "TFA-Twin-Plus-30.3049, Conrad KW9010, Ea2 BL999"
+Registering protocol [91] "Thermopro TP11 Thermometer"
+Registering protocol [92] "Thermopro TP08/TP12/TP20 thermometer"
+Registering protocol [94] "TS-FT002 Wireless Ultrasonic Tank Liquid Level Meter With Temperature Sensor"
+Registering protocol [95] "Visonic powercode"
+Registering protocol [96] "Waveman Switch Transmitter"
+Registering protocol [97] "WG-PB12V1 Temperature Sensor"
+Registering protocol [98] "WS2032 weather station"
+Registering protocol [99] "Hyundai WS SENZOR Remote Temperature Sensor"
+Registering protocol [100] "WT0124 Pool Thermometer"
+Registering protocol [101] "X10 RF"
+Registering protocol [102] "X10 Security"
+```
+
+### Change receive frequency
+
+Default receive frequency of the CC1101 module is 433.92 Mhz, and this can be can changed by sending a message with the frequency.  Parameter is `mhz` and valid values are 300-348 Mhz, 387-464Mhz and 779-928Mhz.  Actual frequency support will depend on your CC1101 board
```

Comment:
Default receive frequency of the module is 433.92 Mhz, and this can be can changed by sending a message with the frequency.  Parameter is `mhz` and valid values are 300-348 Mhz, 387-464Mhz and 779-928Mhz.  Actual frequency support will depend on your board.